ARPG/Assets/Scripts/PlayerStats.cs

```diff
@@ -0,0 +1,86 @@
+using System;
+using UnityEngine;
+
+public class PlayerStats : MonoBehaviour, IDamageable
```

Comment:
Consider renaming PlayerStats to PlayerStatus considering it will not only contain the stats of the player but also status effects such as staggered and eg. poisoned.

Reply:
i am not entirely sure about this as handling of status effects within playerstats was not mentioned at time of setting up the sprint. we can certainly expand the scope to include that as well, of course. if that is requested i'll be glad change the name and add a ticket for it on the trello for the future!

Reply:
I think leaving it as is is fine, I did not request James to have status effects in the player stats script on the trello, and having them as different scripts that can be enabled and disabled on the player (as well as enemies) seem preferable to intergrating them as functions in the player stats script